__tests__/new/hathorwallet.test.ts

```diff
@@ -1371,3 +1372,43 @@ test('setExternalTxSigningMethod', async () => {
   hwallet.setExternalTxSigningMethod(async () => {});
   expect(hwallet.isSignedExternally).toBe(true);
 });
+
+test('getUtxosForAmount - should always get the best utxos', async () => {
+  const seed =
+    'upon tennis increase embark dismiss diamond monitor face magnet jungle scout salute rural master shoulder cry juice jeans radar present close meat antenna mind';
```

Comment:
question(non-blocking): Should we have this seed extracted to a constant somewhere?